shared/build.gradle.kts

```diff
@@ -23,8 +24,14 @@ android {
 }
 
 kotlin {
+    val iOSTarget: (String, KotlinNativeTarget.() -> Unit) -> KotlinNativeTarget =
+        if (System.getenv("SDK_NAME")?.startsWith("iphoneos") == true)
+            ::iosArm64
+        else
+            ::iosX64
```

Comment:
This should be handled with [Hierarchical Project Structure]({android=target android (androidJvm), iosArm64=target iosArm64 (native), iosX64=target iosX64 (native), metadata=target metadata (common)}) but when I look for Kotlin targets in `packForXcode` task, I still see native targets:
```
{android=target android (androidJvm), iosArm64=target iosArm64 (native), iosX64=target iosX64 (native), metadata=target metadata (common)}
```

This might be a problem with documentation or I don't understand something. I plan to create an issue for that on KMM documentation.